sslMode=strict completely broken on 0.1.52

After update to the latest (0.1.52) version we got 
`Caused by: javax.net.ssl.SSLPeerUnverifiedException: Certificate for <www.domain.com> doesn't match any of the subject alternative names: [www.domain.com]` when using sslMode=strict.

The problem is in that PR #295 and if to be more precious in the wrong import:
```java
import static javax.net.ssl.HttpsURLConnection.getDefaultHostnameVerifier;
```
`HttpsURLConnection.getDefaultHostnameVerifier()` returns false on any verification request (see implementation below):
```java
private static class DefaultHostnameVerifier implements HostnameVerifier {
        private DefaultHostnameVerifier() {
        }

        public boolean verify(String domain, SSLSession session) {
            return false;
        }
    }
```

I guess what proper import is 
```java
import static org.apache.http.conn.ssl.SSLConnectionSocketFactory.getDefaultHostnameVerifier();
```

--- a/src/main/java/ru/yandex/clickhouse/util/ClickHouseHttpClientBuilder.java
+++ b/src/main/java/ru/yandex/clickhouse/util/ClickHouseHttpClientBuilder.java
@@ -47,9 +47,6 @@ import java.util.Iterator;
 import java.util.List;
 import java.util.concurrent.TimeUnit;
 
-import static javax.net.ssl.HttpsURLConnection.getDefaultHostnameVerifier;
-
-
 public class ClickHouseHttpClientBuilder {
 
     private final ClickHouseProperties properties;
@@ -75,7 +72,7 @@ public class ClickHouseHttpClientBuilder {
           .register("http", PlainConnectionSocketFactory.getSocketFactory());
 
         if (properties.getSsl()) {
-            HostnameVerifier verifier = "strict".equals(properties.getSslMode()) ? getDefaultHostnameVerifier() : NoopHostnameVerifier.INSTANCE;
+            HostnameVerifier verifier = "strict".equals(properties.getSslMode()) ? SSLConnectionSocketFactory.getDefaultHostnameVerifier() : NoopHostnameVerifier.INSTANCE;
             registry.register("https", new SSLConnectionSocketFactory(getSSLContext(), verifier));
         }
 

